src/mme-app/actionHandlers/attachActionHandlers.cpp

```diff
@@ -38,11 +54,99 @@
 #include <utils/mmeCauseUtils.h>
 #include "gtpCauseTypes.h"
 
+
+
+#define MCC_MNC_LEN 4
```

Comment:
Avoid these defines like this.. What's the purpose of this ? Who is using it..and if not many users then better to avoid.

Reply:
This is used to calculate MNC/MCC. It is used at line 1511 in  ActStatus ActionHandlers::req_to_dns(ControlBlock& cb) function